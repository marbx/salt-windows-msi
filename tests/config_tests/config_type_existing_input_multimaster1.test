properties START_MINION="" MASTER=master.cli1,master.cli2
dormant
