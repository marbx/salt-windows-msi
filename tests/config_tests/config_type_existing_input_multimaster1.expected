# User config line 1/3
master:
- master.cli1
- master.cli2
# User config line 2/3
id: id.config_multimaster1
# User config line 3/3
